### DISCUSSION TITLE
Meldingsboks og alert message - optisk plassering av ikoner

### DISCUSSION THREAD

**Discussion by kristineSorensen** [fremtind/jokul#2602] [Annet]
Designerne i seOpp jobber med denne komponenten nå og vi syns plasseringen av ikonene og Xen ser skjevt ut - mulig de burde sentreres på midten fremfor i bunn. Ekstra tydelig på mobil siden de kommer nærme hverandre. Også rart med X som ikon på error siden det da blir 2 Xer. 

<img width="895" alt="Skjermbilde 2021-12-15 kl  13 28 20" src="https://user-images.githubusercontent.com/62281382/146186758-8e02439a-c04c-46a3-8949-9726efe028d7.png">

**Comment by ivnai** [comment]
Ja eagle eye Kristine, er enig at Xene er litt for lavt på kompakt komponentene, både message box og system message. Om paddingen til toppen kunne bli 2 px mindre, ser dette da godt ut. For min del er det greit å vise 2x X fordi det er nok visuell forskjell mellom ikon og handling, handlingen ligger på typisk plass hvor man forventer den, og mange løsningene bruker liknende så tror ikke dette er uvanlig. Nysgjerrig på hva andre synes.

**Comment by wkillerud** [comment]
Enig i at lukk-knappen kan justeres litt opp så den kommer mer på linje med de andre. Men er enig med Ivana, synes ErrorMessageBox fungerer slik den er.

Jeg kan lage et issue (#2623) på å justere knappeposisjonen :)

**Comment by onordbo** [comment]
Ser ut som det er noe alignment issues jevnt over i MessageBox, tar en kikk på det. Men trenger vi egentlig ikonet i compact varianten? Ref [designprinsipp #3](https://jokul.fremtind.no/profil/designprinsipper#:~:text=klarhet%20gjennom%20%C3%A5%20forenkle%2C%20fjerne%20un%C3%B8dvendig%20st%C3%B8y) fra profilen vår

> ... skaper klarhet gjennom å forenkle, fjerne unødvendig støy ...

Når det er sagt, så ser skjermbildet ovenfor litt forskjellig ut fra hvordan MessageBox fremstår i biblioteket. Har dere satt inn komponenten direkte og ikke overstyrt innstillingene?

**Reply by wkillerud** [reply]
Vil tro vi trenger noe annet enn farge for å indikere viktigheten av meldingen. Og dette er nok `AlertMessage` – visuelt ganske lik, men forskjellige komponenter.

**Comment by onordbo** [comment]
Tok en kikk i Figma-branchen. Jeg synes x'en fortsatt henger litt langt ned rent optisk, relativt ikonene. I tillegg, ved nærmere inspeksjon så ser ikonene litt for store ut i compact versjonen relativt til hele containeren 🤔 Det må nok ryddes litt i hvordan ikonene er gruppert og spacet med Auto-Layout for å få dette til å sitte bra. Det ser ut til at det har blitt brukt litt kreativ plassering innad i containerene for hvert ikon, og det skaper følgefeilene vi nå ser her.

**Reply by onordbo** [reply]
Også ser det ut som master-komponenten for ikonene som brukes i MessageBox er linket til en skisse fra VD-forum.fig?
